releases/release-1.20/release-notes/maps/pr-94903-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 94903
+releasenote:
+  text: Both apiserver_request_duration_seconds metrics and RequestReceivedTimestamp fields of an audit event now take into account the time a request spends in the apiserver request filters.
```

I recommend to quote as much as possible as code:
```suggestion
  text: Both `apiserver_request_duration_seconds` metrics and `RequestReceivedTimestamp` fields of an audit event now take into account the time a request spends in the apiserver request filters.
```